.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+            node-version: 12.x
```

Comment:
why only node 12 and not node 14 ?

Reply:
We have a to set a specific environment for our command to run, as we are moslty using NodeJS 12.x right now, I did choose to go with it.
The command result / behavior should not change in between multiple NodeJS versions; yet I can add a matrix to make it run both versions, if needed?